config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_bigqueryanalyticshublistings.bigqueryanalyticshub.cnrm.cloud.google.com.yaml

```diff
@@ -43,81 +39,73 @@ spec:
     name: v1alpha1
     schema:
       openAPIV3Schema:
+        description: BigQueryAnalyticsHubListing is the Schema for the BigQueryAnalyticsHubListing
+          API
         properties:
           apiVersion:
-            description: 'apiVersion defines the versioned schema of this representation
+            description: 'APIVersion defines the versioned schema of this representation
               of an object. Servers should convert recognized schemas to the latest
-              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
             type: string
           kind:
-            description: 'kind is a string value representing the REST resource this
+            description: 'Kind is a string value representing the REST resource this
               object represents. Servers may infer this from the endpoint the client
-              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
             type: string
           metadata:
             type: object
           spec:
+            description: BigQueryAnalyticsHubListingSpec defines the desired state
+              of BigQueryAnalyticsHubDataExchangeListing
             properties:
-              bigqueryDataset:
```

Comment:
Is this because we dropped Source?  This seems problematic, we can likely get away with it because alpha, but can we avoid the risk?

Reply:
Re-added the union field `source`